fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/synchronizedsession/SynchronizedSessionRunner.java

```diff
@@ -154,7 +160,9 @@ public UUID getSessionId() {
     public CompletableFuture<Void> endSessionAsync() {
         // Using the underlying runner rather than itself because it should not throw any exception if the session
         // does not hold the lock.
-        return underlying.runAsync(context -> session.releaseLock(context.ensureActive()));
+        return underlying.runAsync(context -> session.releaseLock(context.ensureActive()),
+                Lists.newArrayList(LogMessageKeys.TRANSACTION_NAME, "SynchronizedSessionRunner::endSession",
+                        LogMessageKeys.SESSION_ID, session.getSessionId()));
```

Comment:
Should the `SESSION_ID` get added to the user-supplied list of additional log message keys in this runner's implementation of `runAsync` and friends?

Reply:
Good thinking. Done.